problems/dev-environment/index.js

```diff
@@ -1,31 +1,13 @@
 // Old-sk00l multi-line strings
-exports.problem = function () { /*
-One of the most important things that npm does is install packages.
-
-However, in order to be a good workshop program, we don't want to litter
-files all over your computer, so before going any further, let's set up a
-development environment.
-
-First, make a new directory and `cd` into it.
-
-Then, run `npm init` to create a package.json file.  (For extra credit, set
-it up as a git repo as well.)
-
-Run `$ADVENTURE_COMMAND verify` once you're done.  All the other commands
-you run in this tutorial should be done in that folder.
-*/}.toString().split('\n').slice(1,-1).join('\n')
-
-//exports.solution = function () {/*
-//mkdir test
-//cd test
-//npm init
-//# fill in all the prompts
-//
-//# Extra Credit:
-//git init
-//*/}.toString().split('\n').slice(1,-1).join('\n')
+exports.init = function (workshopper) {
+  var postfix = workshopper.lang === 'en' ? '' : '_' + workshopper.lang
+  this.problem = function () {
```

Comment:
That's good idea, but let me do it as a separated another patch to keep the diff more less